pages/projects/projects-spacetimecube.tsx

```diff
@@ -102,6 +121,46 @@ const ProjectSpaceTimeCube: NextPage<Props> = ({
       });
     },
   );
+  const options = {
+    responsive: true,
+    plugins: {
+      legend: {
+        position: "top" as const,
```

Comment:
I think we do not need a legend if we always show only one year. What do you think?

Reply:
Ok, i will delete the legend and without the legend do make the chart more concise.